packages/ember-data/lib/system/relationships/state/relationship.js

```diff
@@ -216,5 +219,9 @@ Relationship.prototype = {
 
   setHasData: function(value) {
     this.hasData = value;
+  },
+
+  setHasLoaded: function(value) {
+    this.hasLoaded = value;
```

Comment:
Great. Do you mind adding a comment to the source to explain the difference for future readers?